### PR TITLE
Continue work on #100

### PR DESCRIPTION
**Pull request by Udera**
I continued the work of #100

### CODE REVIEW

**Comment by Udera** [comment]
@Kami-no @rimas-kudelis : I fixed some small details in the html-code and some minor php errors.

**Comment by rimas-kudelis** [comment]
Gosh, I commented on a cancelled PR. Please check my comments on #122, and why would you create two PR's anyway?

**Comment by Udera** [comment]
Ok, I removed the part of disabling the sa-settings if the sa is disable for the domain. I will put the modifications of the exim-routers in a separate pull request.

File: vexim/sitechangesubmit.php [hunk 1]
```diff
@@ -1,99 +1,137 @@
...
+}
 
-# Just-in-case catchall
-header ("Location: site.php?failupdated={$_POST['domain']}");
-?>
-<!-- Layout and CSS tricks obtained from http://www.bluerobot.com/web/layouts/ -->
+if ($success) {
+  header("Location: site.php?updated={$_POST['domain']}");
+  die;
+} else {
```
Comment (rimas-kudelis) [inline vexim/sitechangesubmit.php]: Like I said before, this `else` is unnecessary.

Comment (rimas-kudelis) [inline vexim/sitechangesubmit.php]: Nevermind though.
I trust there's no need for me to review everything else again, so I've only reviewed the bits where I had remarks previously. Merging this in.

**Comment by rimas-kudelis** [comment]
I trust there's no need for me to review everything else again, so I've only reviewed the bits where I had remarks previously. Merging this in.

**Pull request merged**